### ISSUE TITLE
Different options generated

### ISSUE THREAD

**Issue by avalladaresm** [sequelize/sequelize-auto#408]
So I used sequelize-auto for the first time about 3 weeks ago to generate a couple of models from my db, but unfortunately I don't remember the exact version I used, probably the last one by then, which was 0.4.29, because I just noticed it was then updated 10 days ago and then 3 days ago.
Anyway, 3 weeks ago, I ran the `sequelize-auto` command and a model was generated exactly like the following:
```
const {
  DataTypes
} = require('sequelize')

module.exports = sequelize => {
  const attributes = {
    id: {
      type: DataTypes.STRING(2),
      allowNull: false,
      defaultValue: null,
      primaryKey: true,
      autoIncrement: false,
      comment: null,
      field: 'id',
      unique: 'UQ__Countrie__3213E83E0E52B734'
    },
    name: {
      type: DataTypes.STRING(255),
      allowNull: false,
      defaultValue: null,
      primaryKey: false,
      autoIncrement: false,
      comment: null,
      field: 'name',
      unique: 'UQ__Countrie__72E12F1BC4AAC3FF'
    }
  }
  const options = {
    tableName: 'Countries',
    comment: '',
    indexes: [],
    timestamps: false,
  }
  const CountriesModel = sequelize.define('Countries_model', attributes, options)
  return CountriesModel
}
```
I added a couple of endpoints on my controller and everything worked wonderfully.
Today I needed to generate a new model but the way it was generated was a bit different: 
```
/* jshint indent: 2 */

module.exports = function(sequelize, DataTypes) {
  return sequelize.define('Products', {
    id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      primaryKey: true
    },
    name: {
      type: DataTypes.STRING(255),
      allowNull: false
    },
    inStock: {
      type: DataTypes.BOOLEAN,
      allowNull: false,
      defaultValue: 0
    },
    qtyInStock: {
      type: DataTypes.INTEGER,
      allowNull: false
    },
    code: {
      type: DataTypes.STRING(255),
      allowNull: false
    },
    description: {
      type: DataTypes.STRING(255),
      allowNull: false
    },
    price: {
      type: DataTypes.DECIMAL,
      allowNull: false
    },
    image: {
      type: DataTypes.STRING(255),
      allowNull: true
    },
    categoryId: {
      type: DataTypes.STRING(255),
      allowNull: false,
      references: {
        model: {
          tableName: 'Categories',
        },
        key: 'id'
      }
    }
  }, {
    sequelize,
    tableName: 'Products',
    schema: 'dbo'
  });
};
```
No arrow function, no `field` or `unique` properties on the columns objects.
I executed one endpoint but got an error message: `{ "message": "Cannot read property 'INTEGER' of undefined"}`
Then I created an `attributes` and `options` object just as the model generated 3 weeks ago, but with out changing the columns objects, the result being: 
```
/* jshint indent: 2 */
const {
  DataTypes
} = require('sequelize')

module.exports = sequelize => {
  const attributes = {
    id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      primaryKey: true
    },
    name: {
      type: DataTypes.STRING(255),
      allowNull: false
    },
    inStock: {
      type: DataTypes.BOOLEAN,
      allowNull: false,
      defaultValue: 0
    },
    qtyInStock: {
      type: DataTypes.INTEGER,
      allowNull: false
    },
    code: {
      type: DataTypes.STRING(255),
      allowNull: false
    },
    description: {
      type: DataTypes.STRING(255),
      allowNull: false
    },
    price: {
      type: DataTypes.DECIMAL,
      allowNull: false
    },
    image: {
      type: DataTypes.STRING(255),
      allowNull: true
    },
    categoryId: {
      type: DataTypes.STRING(255),
      allowNull: false,
      references: {
        model: {
          tableName: 'Categories',
        },
        key: 'id'
      }
    }
  }
  const options = {
    tableName: 'Products',
    schema: 'dbo',
    timestamps: false
  }
  const ProductsModel = sequelize.define('Products_model', attributes, options)
  return ProductsModel
};
```
Afterwards, I executed the endpoint and it returned the data I needed. It seems that the update broke something maybe? Or maybe I'm missing something. If you could help me how to generate the model just as the first example here, that would be greatly appreciated!

**Comment by steveschmitt** [comment]
I'll look into it.  Can you please tell me:
1. What dialect you are using?
2. What version of `sequelize` are you using?
3. What command-line options did you use?
4. How do you import the models into your application?

**Comment by avalladaresm** [comment]
Hey, thank you for looking into it.
1. mssql
2. "sequelize": "^6.1.0",
3. `./node_modules/sequelize-auto/bin/sequelize-auto -o "./models" -d sequelize_auto_test -h localhost -u my_username -p 1433 -x my_password -e mssql` (./node_modules/sequelize-auto/bin/sequelize-auto because using only `sequelize-auto` wasn't working, error: `sequelize-auto: command not found`)
4. Since my models are in a ./models directory, on my controller file I simply `var ProductsModel = require('../models/Products')`

**Comment by steveschmitt** [comment]
The code produced by `sequelize-auto` is a definition function for the model.  So you need to `require` and also call the function: 

    var ProductsModel = require('../models/Products')(sequelize, DataTypes);

**Comment by avalladaresm** [comment]
You're right, I missed that! I was only passing  `(sequelize)` instead of `(sequelize, DataTypes)` when using the findAll() function. What about the other properties like 
```
```

**Comment by steveschmitt** [comment]
`comment` only appears when it is not null.
`field` only appears when it is not the same as the property name
I'm not sure about the `unique` value; I thought that was only supposed to be `true` or `false`!

**Comment by avalladaresm** [comment]
I see, well it works just fine that way it is now, so that's a good thing, thanks!

**Comment by steveschmitt** [comment]
Fixed problem with `unique` in 0.6.1
